shepherd/Utils.py

```diff
@@ -0,0 +1,7 @@
+
+# Values for the cooldowns
```

Comment:
Wrap this into a class (e.g. CONSTANTS or SHEPHERD_CONSTANTS). when calling from classes use from shepherd import *, and use CONSTANTS.CONSTANT_NAME